src/main/java/com/hei/wallet/wallety/endpoint/rest/model/Transfert.java

```diff
@@ -0,0 +1,21 @@
+package com.hei.wallet.wallety.endpoint.rest.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Transfert {

```

Comment:
Read the api spec... Transfer is the same transfer there ... like createdAt not created_at in there...